pkg/cmd/chat/channeltype/channeltype.go

```diff
@@ -0,0 +1,214 @@
+package channeltype
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	stream "github.com/GetStream/stream-chat-go/v5"
+	"github.com/GetStream/stream-cli/pkg/config"
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+)
+
+func NewCmds() []*cobra.Command {
+	return []*cobra.Command{
+		getCmd(),
+		createCmd(),
+		deleteCmd(),
+		updateCmd(),
+		listCmd()}
+}
+
+func getCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get-channel-type --channel-type [channel-type] --output-format [json]",
+		Short: "Get channel type",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c, err := config.GetConfig(cmd).GetStreamClient(cmd)
```

Comment:
slowly drop prefix such that `GetClient`

Reply:
✅